hotline/templates/users/user_detail.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html' %}
+{% block content %}
+<div class="pull-right">
+    <a class="btn btn-success" href="{% url 'users-home' %}">Home</a>
+    <a class="btn btn-success" href="{% url 'users-detail' user.pk %}">View Profile</a>
+    <a class="btn btn-success" href="{% url 'users-edit' user.pk %}">Edit Profile</a>
+    <a class="btn btn-success" href="{% url 'reports-list' %}">Search Reports</a>
+    <a class="btn" href="{% url 'logout' %}">Logout</a>
+</div>
+
+    <h3>Details</h3>
+    <p>Name: {% if object.prefix != "None" %}
```

Comment:
Do you mean to compare against None (as opposed to the string "None"?). I don't think it will ever be None though. It could be the empty string though.